db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/cleanup.rb

```diff
@@ -12,6 +12,9 @@ def cleanup
       $evm.log(:info, "deleting #{RHEL_SUBSCRIBE_INVENTORY}")
       system "sudo rm #{RHEL_SUBSCRIBE_INVENTORY}"
     end
+    if Process.kill(0, $evm.root['agent_pid']) == 1
+      system({"SSH_AGENT_PID" => $evm.root['agent_pid']}, "(ssh-agent -k) &> /dev/null")
```

Comment:
1 - We do not make sure the process is actually gone 
2 - we need to handle it even when Process.kill throws an exception